third_party/bigframes_vendored/pandas/plotting/_core.py

```diff
@@ -46,3 +48,248 @@ def hist(self, by: Optional[Sequence[str]] = None, bins: int = 10, **kwargs):
+        Use the `sampling_random_state` parameter to modify the sampling seed.
+
+        Parameters
+        ----------
+        x : label or position, optional
+            Coordinates for the X axis. By default uses the index.
+        y : label or position, optional
+            Column to plot. By default uses all columns.
+        stacked : bool, default True
+            Area plots are stacked by default. Set to False to create a
+            unstacked plot.
+        sampling_n: int, default 100:
+            Number of random items for plotting.
+        sampling_random_state: int, default 0:
+            Seed for random number generator.
+        **kwargs
+            Additional keyword arguments are documented in
+            :meth:`DataFrame.plot`.
+
+        Returns
+        -------
+        matplotlib.axes.Axes or numpy.ndarray
+            Area plot, or array of area plots if subplots is True.
+
+        Examples
+        --------
+        Draw an area plot based on basic business metrics:
+
+        .. plot::
+            :context: close-figs
+
+            >>> import bigframes.pandas as bpd
+            >>> df = bpd.DataFrame(
+            ...     {
+            ...         'sales': [3, 2, 3, 9, 10, 6],
+            ...         'signups': [5, 5, 6, 12, 14, 13],
+            ...         'visits': [20, 42, 28, 62, 81, 50],
+            ...     },
+            ...     index=["01-31", "02-28", "03-31", "04-30", "05-31", "06-30"]
+            ... )
+            >>> ax = df.plot.area()
+
+        Area plots are stacked by default. To produce an unstacked plot,
+        pass ``stacked=False``:
+
+        .. plot::
+            :context: close-figs
+
+            >>> ax = df.plot.area(stacked=False)
+
+        Draw an area plot for a single column:
+
+        .. plot::
+            :context: close-figs
+
+            >>> ax = df.plot.area(y='sales')
+
+        Draw with a different `x`:
+
+        .. plot::
+            :context: close-figs
+
+            >>> df = bpd.DataFrame({
+            ...     'sales': [3, 2, 3],
+            ...     'visits': [20, 42, 28],
+            ...     'day': [1, 2, 3],
+            ... })
+            >>> ax = df.plot.area(x='day')
+        """
+        raise NotImplementedError(constants.ABSTRACT_METHOD_ERROR_MESSAGE)
+
+    def scatter(
+        self,
+        x: typing.Optional[typing.Hashable] = None,
+        y: typing.Optional[typing.Hashable] = None,
+        s: typing.Union[typing.Hashable, typing.Sequence[typing.Hashable]] = None,
+        c: typing.Union[typing.Hashable, typing.Sequence[typing.Hashable]] = None,
+        **kwargs,
+    ):
+        """
+        Create a scatter plot with varying marker point size and color.
+
+        This function calls `pandas.plot` to generate a plot with a random sample
+        of items. For consistent results, the random sampling is reproducible.
+        Use the `sampling_random_state` parameter to modify the sampling seed.
+
+        Parameters
```

Comment:
Should we convert these docstrings to google style? eg `Args:` instead of `Parameters`